### PR TITLE
Add short hash view style

### LINKED ISSUE #2: Truncated hashes
It would be nice to have an option/toggle that git-style truncates
`/nix/store/ghzg4kg0sjif58smj2lfm2bdvjwim85y-gcc-wrapper-7.4.0`
to e.g.
`ghzg4k..-gcc-wrapper-7.4.0`

Excuse me closing the issue; I initially overlooked the "View" box, so I closed the issue when I saw you had similar features already, and then reopened when I realized I wanted the truncated hash anyway

### PR DESCRIPTION
**Pull request by jonascarpay**
Closes #2

### CODE REVIEW

File: src/nix_query_tree.rs [hunk 1]
```diff
@@ -56,6 +56,34 @@ impl NixQueryDrv {
...
+                            }
+                            Some(short_hash) => {
+                                format!("{}..{}", short_hash, drv_name)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
```
Comment (jonascarpay) [inline src/nix_query_tree.rs]: Maybe there's a cleaner way to do this *cough*

Comment (jonascarpay) [reply src/nix_query_tree.rs]: turns out there was

File: src/nix_query_tree.rs [hunk 2]
```diff
@@ -56,6 +56,33 @@ impl NixQueryDrv {
...
+    /// let nix_query_drv =
+    ///     NixQueryDrv::from("/nix/store/az4kl5slhbkmmy4vj98z3hzxxkan7zza-gnugrep-3.3");
+    /// assert_eq!(
+    ///     nix_query_drv.short_hash_and_drv_name(),
+    ///     String::from("az4kl5s..gnugrep-3.3")
+    /// );
+    /// ```
+    pub fn short_hash_and_drv_name(&self) -> String {
+        let drv_str = self.0.to_string_lossy();
+        let drv_str_no_store = String::from((drv_str).trim_start_matches("/nix/store/"));
```
Comment (cdepillabout) [inline src/nix_query_tree.rs]:
```suggestion
        let drv_str_no_store = String::from(drv_str.trim_start_matches("/nix/store/"));
```

**Comment by cdepillabout** [comment]
LGTM

Thanks for taking the time to add this.

**Pull request merged**